go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/tidwall/sjson
 
 go 1.14
 
+replace github.com/tidwall/sjson => github.com/ContinuumLLC/sjson v1.2.5
```

Comment:
What is the purpose of `sjson` replacement as we should not have a dependency on the same project?

Reply:
We do replacement to specify concrete version of the source library. You can take a look to other repos that we have cloned, they also have the same thing, for example https://github.com/ContinuumLLC/hystrix-go/blob/master/go.mod#L13